src/WeekComponent/WeekComponent.js

```diff
@@ -119,25 +119,38 @@ class WeekComponent extends React.Component {
     }
   }
 
+  changeTimer () {
```

Comment:
the name changeTimer is not descriptive of what this function does. Something like `switchCountingUp` would be more appropriate.